alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+      keywords:
+        description: Keywords associated with a transition
+        type: array
+        items:
+          type: string
+        example:
+          - keyword1
+          - keyword2
+      properties:
+        description: Key-value-pair metadata associated with a transition
+        type: object
+        additionalProperties:
+          type: string
+        example:
+          key1: value1
+  SubQuery:
+    title: Nested Query Body
+    type: object
+    properties:
+      alarmId:
+        type: string
+        example: System.Health.DiskSpaceLow.Bdd9u!4aMd!$pYrf*CnaIZ2tbu$-Ct%?
+      displayName:
+        type: string
+        description: Display name query. The service will return alarm instances whose *displayName* fields contain the display name query string, without taking case into account.
+        example: Low disk space on CRIO1
+      active:
+        type: boolean
+        example: true
+      clear:
+        type: boolean
+        example: false
+      acknowledged:
+        type: boolean
+        example: false
+      acknowledgedAtMin:
+        type: string
+        description: Acknowledged at minimum date query. The service will return instances which were acknowledged between *acknowledgedAtMin* and *acknowledgedAtMax*, inclusive.
+        example: '2019-01-14T23:15:53.7270539Z'
+      acknowledgedAtMax:
+        type: string
+        description: Acknowledged at maximum date query. The service will return instances which were acknowledged between *acknowledgedAtMin* and *acknowledgedAtMax*, inclusive.
+        example: '2019-01-15T23:15:53.7270539Z'
+      occurredAtMin:
+        type: string
+        description: Occurred at minimum date query. The service will return instances which occurred between *occurredAtMin* and *occurredAtMax*, inclusive.
+        example: '2019-01-14T23:15:53.7270539Z'
+      occurredAtMax:
+        type: string
+        description: Occurred at maximum date query. The service will return instances which occurred between *occurredAtMin* and *occurredAtMax*, inclusive.
+        example: '2019-01-15T23:15:53.7270539Z'
+      currentSeverityLevelMin:
+        type: integer
+        description: Current severity minimum query. The service will return instances whose current severity is between *currentSeverityLevelMin* and *currentSeverityLevelMax*, inclusive.
+        example: 1
+      currentSeverityLevelMax:
+        type: integer
+        description: Current severity maximum query. The service will return instances whose current severity is between *currentSeverityLevelMin* and *currentSeverityLevelMax*, inclusive.
+        example: 4
+      highestSeverityLevelMin:
+        type: integer
+        description: Highest severity minimum query. The service will return instances whose highest severity is between *highestSeverityLevelMin* and *highestSeverityLevelMax*, inclusive.
+        example: 1
+      highestSeverityLevelmax:
```

Comment:
highestSeverityLevelmax -> highestSeverityLevel**M**ax

Reply:
Woops! I confess to not having tested all of the query properties.